src/ClientApp/src/translations/de/common.json

```
@@ -25,5 +25,13 @@
   "model-preview": "Modellvorschau",
   "invalid-model-url": "Ungültige Modell-URL",
   "no-model-preview": "Keine Modellvorschau verfügbar",
-  "model-uri": "Modell-URI"
+  "model-uri": "Modell-URI",
+  "width": "Breite",
+  "height": "Höhe",
+  "generate-embed-tag": "IFrame zum Einbetten in externe Website generieren",
```

PP: Korrekte Schreibweise von iFrame verwenden und Webseite anstatt Website:
```suggestion
  "generate-embed-tag": "iFrame zum Einbetten in externe Webseite generieren",
```